### DISCUSSION TITLE
Custom beep?

### DISCUSSION THREAD

**Discussion by aguLepe** [IDI-Systems/acre2#1123] [Q&A] [answered]
Love ACRE2! Would be nice to be able to customize radio's beep. Is there any way to do this?

**Comment by PabstMirror** [comment, marked as answer]
sounds are defined in https://github.com/IDI-Systems/acre2/blob/master/addons/sys_sounds/CfgAcreSounds.hpp
to make your own you can just create an addon that requires `acre_sound`
and then redefine those configs, use `Wav2B64.exe in.wav` (it should be in the \extra folder of an acre release)

**Reply by aguLepe** [reply]
Thanks for your reply. I tried to use that .exe. Sincerely, i have no idea how to make it work. Executing the .exe  is not working. Drag a .wav to .exe is not working either. Do you have some information to use that tool?

Edit---
Well, for some unknown reason, wav2b64.exe works fine with ACRE2 .wav original files. For any other .wav files that I download, is not working. Have no idea about the difference between ACRE2 Original .wav files and .wav files that I download.

**Reply by jonpas** [reply]
It's a command line application. Then you will also see the output and errors.

Has to be mono, 16-bit PCM, 48 kHz.

**Reply by aguLepe** [reply]
Your info is completely right. With that parameters wav2b64 is working without problems, thanks for your help!

**Comment by maxcloutier13** [comment]
> sounds are defined in https://github.com/IDI-Systems/acre2/blob/master/addons/sys_sounds/CfgAcreSounds.hpp to make your own you can just create an addon that requires `acre_sound` and then redefine those configs, use `Wav2B64.exe in.wav` (it should be in the \extra folder of an acre release)

So I managed to override the sound using this. Is there a way to have a different beep for different radios? I would like to have it more obvious when you are calling on the 343 vs the LR radios. I have not seen an obvious way to do this...

**Reply by jonpas** [reply]
That is sadly hard-coded for all radios. We could potentially change that, maybe just adding separate `CfgAcreSounds` config for each radio could work without additional overhead, then you could override per-radio. PRs welcome.

**Reply by maxcloutier13** [reply]
While I am starting to understand the internal functioning of the mod I don't think I am able to participate in such a way sadly... Not for now at least... 
But thanks for confirming I was not insane! Looking for where to set this led me in the rabbit hole hehe.

**Reply by PabstMirror** [reply]
I can take a look at this, I'm thinking of just adding a config to the `CfgAcreComponents`

**Reply by maxcloutier13** [reply]
Let me know if I can help.
